File: ToughDashApr19/two.c
#include<stdio.h>

int main() {
    unsigned long int num, rem, count[10];

    scanf("%lu", &num);

    for(unsigned int i=0;i<10;i++)
        count[i] = 0;
    
    while(num > 0) {
        rem = num%10;
        num = num / 10;

        count[rem-1]++;
    }

    unsigned int max = count[0], maxIndex = 0;

    for(unsigned int i=1;i<10;i++) {
        if(count[i] > max) {
            max = count[i];
            maxIndex = i;
        }
    }

    printf("%u", maxIndex+1);

    
    return 0;
}

File: ToughDashApr19/two.py
counter = [0 for x in range(10)] # initializing list

num = int(input())

while num != 0:
    rem = num % 10
    num = num // 10 # Integer division/floor division
    counter[rem-1] = counter[rem-1] + 1

maxIndex, max = -1, -1

for i, x in enumerate(counter): # for getting index of loop
    if x > max:
        max = x
        maxIndex = i

print(maxIndex+1)